### ISSUE TITLE
Fails to install any version of Opentofu due to SHA256SUMS signature mismatch

### ISSUE THREAD

**Issue by edtep** [tofuutils/tofuenv#53]
**Describe the bug**
When trying to install Opentofu using tofuenv, the installation fails:
```console
$ tofuenv install
Installing OpenTofu v1.6.1
Downloading release tarball from https://github.com/opentofu/opentofu/releases/download/v1.6.1/tofu_1.6.1_darwin_arm64.zip
######################################################################### 100.0%
Downloading SHA hash file from https://github.com/opentofu/opentofu/releases/download/v1.6.1/tofu_1.6.1_SHA256SUMS
Downloading SHA hash signature file from https://github.com/opentofu/opentofu/releases/download/v1.6.1/tofu_1.6.1_SHA256SUMS.sig
▶ ERROR openpgp: invalid data: tag byte does not have MSB set
SHA256SUMS signature does not match!
```

The same error occurs if I try to install an older version:
```console
$ tofuenv install 1.6.0
Installing OpenTofu v1.6.0
Downloading release tarball from https://github.com/opentofu/opentofu/releases/download/v1.6.0/tofu_1.6.0_darwin_arm64.zip
######################################################################### 100.0%
Downloading SHA hash file from https://github.com/opentofu/opentofu/releases/download/v1.6.0/tofu_1.6.0_SHA256SUMS
Downloading SHA hash signature file from https://github.com/opentofu/opentofu/releases/download/v1.6.0/tofu_1.6.0_SHA256SUMS.sig
▶ ERROR openpgp: invalid data: tag byte does not have MSB set
SHA256SUMS signature does not match!
```

**To Reproduce**
Run `tofuenv install`.

**Expected behavior**
Running `tofuenv install` should install Opentofu successfully.

**Environment (please complete the following information):**
- OS: macOS Sonoma 14.2.1
- tofuenv version 1.0.3

**Additional context**
I have jq, gnupg and grep installed. Adding 'yes' to use-gpgv did not help.

**Comment by anastasiiakozlova245** [comment]
Hi @edtep, 

Please upgrade tofuenv to the latest 1.0.5, this issue is already fixed.
`brew update`
`brew upgrade tofuenv`

**Comment by edtep** [comment]
Hi @anastasiiakozlova245 

Thank you! I upgraded tofuenv and it works now.

I did notice something odd, though. I installed tofuenv for the very first time yesterday, and had run `brew update` prior to doing so, but received the older version 1.0.3 for some reason. Just now, I removed the installation of tofuenv and re-added the tofuutils tap. When I ran `brew update` and `brew install tofuenv`, Homebrew installed tofuenv in `/opt/homebrew/Cellar/tofuenv/1.0.5` and if I run `brew upgrade tofuenv`, I get `Warning: tofuutils/tap/tofuenv 1.0.5 already installed`, yet `tofuenv -v` shows 1.0.4.
  
The original issue is solved, though, so I'm closing this.
